src/idl_gen_text.cpp

```diff
@@ -51,11 +51,24 @@ bool Print(T val, Type type, int /*indent*/, Type * /*union_type*/,
            const IDLOptions &opts, std::string *_text) {
   std::string &text = *_text;
   if (type.enum_def && opts.output_enum_identifiers) {
-    auto ev = type.enum_def->ReverseLookup(static_cast<int64_t>(val));
-    if (ev) {
-      text += "\"";
-      text += ev->name;
-      text += "\"";
+    auto enum_values = [&type, &val]() -> std::vector<EnumVal const*> {
+      if (auto ev = type.enum_def->ReverseLookup(static_cast<int64_t>(val))) {
+          return {ev};
+      }
+      if (!type.enum_def->attributes.Lookup("bit_flags")) return {};
+      if (!val) return {};
+      std::vector<EnumVal const*> ret;
+      ret.reserve(kTypeSizes[type.enum_def->underlying_type.base_type] * 8);
```

Comment:
this reserve is probably unnecessary since typically few bits are set..

Reply:
ok